lua/lspconfig/jsonnet.lua

```diff
@@ -0,0 +1,22 @@
+local configs = require 'lspconfig/configs'
+local util = require 'lspconfig/util'
+
+configs.jsonnet = {
+  default_config = {
+    cmd = { 'jsonnet-language-server' },
+    filetypes = { 'jsonnet', 'libsonnet' },
+    root_dir = function(fname)
+      return util.root_pattern 'go.work'(fname) or util.root_pattern('go.mod', '.git')(fname)
```

Comment:
This may not be correct.

Reply:
I want this to launch from the directory that vim was launched from.  Does someone have a pointer for this?

Reply:
We're not going to support that in lspconfig until I have time to implement scratch server support.